conman/routes/models.py

```diff
@@ -82,6 +82,27 @@ def handle(self, request, path):
         # Deal with the request
         return handler.handle(request, path)
 
+    def move_to(self, new_url, *, move_children):
```

Comment:
I see what you mean but I think it's clear enough when you read the code. Otherwise here are a few suggestions I could think of are but they are all composed with two words:
 - `move_to(self, new_url, *, include_children)`;
 - `move_to(self, new_url, *, affect_children)`;
 - `move_to(self, new_url, *, update_subpath)`.